File: tuan01/tong.jl
function main()
    lines = readlines(stdin)
    tokens = SubString{String}[]
    for line in lines
        append!(tokens, split(line))
    end
    if isempty(tokens)
        return
    end
    
    n = parse(Int64, tokens[1])
    arr = [parse(Int64, tokens[i]) for i in 2:n+1]
    
    total_sum = sum(arr)
    max_val = maximum(arr)
    
    println("$total_sum $max_val")
end

main()

File: tuan01/tong.py
import sys

def main():
    input_data = sys.stdin.read().split()
    if not input_data:
        return
    
    n = int(input_data[0])
    arr = [int(x) for x in input_data[1:n+1]]
    
    total_sum = sum(arr)
    max_val = max(arr)
    
    print(f"{total_sum} {max_val}")

if __name__ == '__main__':
    main()